modules/web/site/basic-publishing-credentials-policy/main.bicep

```diff
@@ -9,6 +9,9 @@ metadata owner = 'Azure/module-maintainers'
 ])
 param name string
 
+@sys.description('Optional. Set to true to allow access to or false to diable a publishing method.')
```

Comment:
```suggestion
@sys.description('Optional. Set to true to enable or false to disable a publishing method.')
```

Reply:
Note: Wil require an update of the Json and readme

Reply:
@ahmadabdalla updated. please review again thanks.